dGame/dGameMessages/GameMessages.cpp

```diff
@@ -328,7 +328,7 @@ void GameMessages::SendPlayNDAudioEmitter(Entity* entity, const SystemAddress& s
 		bitStream.Write(static_cast<char>(audioGUID[k]));
 	}
 
-	bitStream.Write(uint32_t(0));
+	bitStream.Write(static_cast<uint32_t>(0));
```

Comment:
In this case just remove the static cast and change this line to `bitStream.Write<uint32_t>(0);`

Reply:
there are multiple occurances of this change so this applies to all

Reply:
Managed to convert all but 3, because those 3 need the intermediate step due to being a conversion from an enum